.vscode/launch.json

```diff
@@ -10,15 +10,15 @@
       "request": "launch",
       "mode": "debug",
       "program": "${workspaceRoot}/src",
-      "args": ["--config=${workspaceRoot}/themes/jandedobbeleer.omp.json"]
+      "args": ["-config=${workspaceRoot}/themes/jandedobbeleer.omp.json"]
```

Comment:
why not `--`? that's the way that's "normal"

Reply:
mistake. Should not have been pushed.